frontend/src/components/BottomSignUp/BottomSignUp.js

```diff
@@ -1,10 +1,16 @@
 import React, {Component } from 'react';
 import './BottomSignUp.css'
 import signupImage from "../../assets/soup-kitchen.jpg";
+import { withRouter } from 'react-router'
 
 class BottomSignUp extends Component {
 
+  
+
   render (){
+
+    const { match, location, history } = this.props
```

Comment:
I don't think you need this/are using it right now. Since you specify `this.props.history` below, you don't need to decompose it here. Plus, you're not using match or location at all.